packages/kotti-ui/source/kotti-field-file-upload/components/DropArea.vue

```diff
@@ -0,0 +1,276 @@
+<template>
+	<div
+		ref="dropAreaRef"
+		:class="classes"
+		:data-test="dataTest ? `${dataTest}.dropArea` : undefined"
+		:tabIndex="_tabIndex"
+		@click.prevent="onClick"
+		@dragleave.prevent.stop="isDragging = false"
+		@dragover.prevent.stop="isDragging = true"
+		@drop.prevent.stop="onDrop"
+		@mouseenter.prevent.stop="isHovering = true"
+		@mouseleave.prevent.stop="isHovering = false"
+	>
+		<div class="kt-field-file-upload__drop-area__description">
+			<i class="yoco" v-text="_icon" />
```

Comment:
why is it prop-customizable, do we have a use-case to customise the icon from outside?

Reply:
This was requested by design